src/JsonSchema/SchemaFactory.php

```diff
@@ -65,6 +65,14 @@ public function buildSchema(string $className, string $format = 'json', string $
 
         [$operation, $serializerContext, $validationGroups, $inputOrOutputClass] = $metadata;
```

Comment:
To save you digging too hard, and easing the rebase, after my last PR these are now assigned here:
https://github.com/api-platform/core/blob/678eb4f8474edc29b2fc806c85aaf203540e23ab/src/JsonSchema/SchemaFactory.php#L62-L70